### PR TITLE
Handle structs as return type in C and LLVM backends

### PR DESCRIPTION
**Pull request by czgdp1807**

### CODE REVIEW

File: src/libasr/codegen/asr_to_cpp.cpp [hunk 1]
```diff
@@ -210,8 +210,11 @@ class ASRToCPPVisitor : public BaseCCPPVisitor<ASRToCPPVisitor>
     }
 
     std::string convert_variable_decl(const ASR::Variable_t &v, bool use_static=true,
-                                      bool use_templates_for_arrays=false)
+                                      bool use_templates_for_arrays=false, bool _=false)
     {
+        if( _ ) {
+
+        }
```
Comment (certik) [inline src/libasr/codegen/asr_to_cpp.cpp:217]: What is this for?

I recommend not using default arguments, and just use `bool /* arg_name */`.

**Review by certik — APPROVED** [review]

**Review by certik — APPROVED** [review]
Looks good to me.

**Comment by czgdp1807** [comment]
I will be implementing the actual solution for struct return types which we discussed in #1289. I am converting this to draft for now.

**Comment by czgdp1807** [comment]
Closing in favour of https://github.com/lcompilers/lpython/pull/1295

**Pull request closed without merging**